coralibre-sdk/sdk/src/main/java/org/coralibre/android/sdk/internal/database/ppcp/model/IntervallOfCapturedData.java

```diff
@@ -4,12 +4,12 @@
 
 import java.util.List;
 
-public interface IntervalOfCollectedPackages {
+public interface IntervallOfCapturedData {
```

Comment:
```suggestion
public interface IntervalOfCapturedData {
```

Reply:
xD who do you think I am. Of cource Typos :P

Reply:
No for real I'll fix it.

Reply:
I meant, I still don't have the knowledge to really review the code ;-D